ui/settings.glade

```diff
@@ -1,201 +1,201 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.38.2 -->
+<!-- Generated with glade 3.22.2 -->
```

Comment:
For a smaller diff, consider using a newer version of glade. Version 3.22 uses '_'s in many places where 3.38 uses '-'s. :)